strings/base_array.h

```diff
@@ -511,24 +511,24 @@ WINRT_EXPORT namespace winrt
         return impl::com_array_proxy<T>(__valueSize, value);
     }
 
-    inline hstring get_class_name(Windows::Foundation::IInspectable const& object)
+    inline hstring get_class_name(Windows::Foundation::IInspectable const& object WINRT_IMPL_SOURCE_LOCATION_ARGS)
```

Comment:
Given the `check_hresult` fix I mentioned, we should be able to hold off changing `base_array.h` at all.

Reply:
I think you're right.  I was able to remove this and it still compiles 👍